src/sass/main.scss

```diff
@@ -2,3 +2,64 @@
 
 @import 'base/base';
 @import 'base/utilities';
+
+body{
+    background-color: #f0f0f0;
+}
+
+.header{
+    background-color: white;
+    position: sticky;
+    display: flex;
+    flex-wrap: nowrap;
+    min-height: auto;
+    text-decoration: none;
+
+.logo{
+    height: 60px;
+    width: 40px;
+    margin-left: 20px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+}
+
+.logotext{
+    position: relative;
+    align-self: center;
+    left:20px;
+    font-family: 'Rubik', sans-serif;
+    font-weight: 700;
+    font-size: x-large;
+    text-decoration: none;
+    color: black;
+}
+
+.header-right{
+    margin-left: auto;
+    align-self: center;
+    margin-right: 10px;
+    display: flexbox;
+    
+    .slack{
+        align-self: center;
+        min-width: 120px;
+        height: 40px;
+        background-color: #0ab50a;
```

Comment:
Could you move this color to the variables file? Maybe call it `slack-button-color` or something. On this line, you can reference the scss variable. It just makes it easier to reuse the color, and change it in the future if we need to for whatever reason 😊